overview.html

```diff
@@ -0,0 +1,452 @@
+    attachment data, Base64 encoded, within the document body. The
+    {@link com.cloudant.client.api.model.Attachment} class represents an inline attachment.
+    Classes that extend {@link com.cloudant.client.api.model.Document} automatically inherit
+    support
+    for inline attachments.
+</P>
+
+<P>Example usage of inline attachment, using Apache Commons Codec for Base64 encoding:</P>
+<pre>
+    {@code
+        Attachment attachment = new Attachment();
+        attachment.setData(Base64.encodeBase64String("attachment test string".getBytes()));
+        attachment.setContentType("text/plain");
+        Foo foo = new Foo(); // Foo extends Document
+        foo.addAttachment("attachment.txt", attachment);
+        db.save(foo);
+    }
+</pre>
+<P>Note that attachment data is <strong>not</strong> included by default when reading documents.
+    To
+    retrieve inline attachment data use {@link
+    com.cloudant.client.api.model.Params#attachments()}.
+    Seeh {@link com.cloudant.client.api.Database#find(java.lang.Class, java.lang.String,
+    com.cloudant.client.api.model.Params)} for an example.
+</P>
+
+<h2 id="Design Document Functions">Design Document Functions</h2>
+
+<P>
+    These functions are for working with views and design documents, including querying the database
+    using map-reduce views, <a href="#Cloudant Search">Cloudant Search</a>, and
+    <a href="#Cloudant Query">Cloudant Query</a>. It is recommend to read the
+    <a href="https://docs.cloudant.com/design_documents.html">Cloudant design document documentation
+    </a> before working with design documents to gain an understanding of the available parameters
+    and funtions.
+</P>
+
+<P>
+    The {@link com.cloudant.client.api.model.DesignDocument} class encapsulates a design document in
+    a Java object.
+</P>
+
+<P>See {@link com.cloudant.client.api.DbDesign#getFromDb(java.lang.String)} for an example of
+    retrieving a design document from the sever.</P>
+
+<P>Example of creating a view (map-reduce index) using {@link java.util.Map} to generate
+    the JSON for the design document </P>
+<pre>
+    {@code
+    // Uploads the design document with view index:
+    // {
+    //   "_id": "_design/name",
+    //   "views": {
+    //     "view1": {
+    //       "map":"function(doc){emit(doc.field, 1)}",
+    //       "reduce": "function(key, value, rereduce){return sum(values)}"
+    //     }
+    //   }
+    // }
+
+    Map<String, Object> view1 = new HashMap<>();
+    view1.put("map", "function(doc){emit(doc.field, 1)}");
+    view1.put("reduce", "function(key, value, rereduce){return sum(values)}");
+
+    Map<String, Object> views = new HashMap<>();
+    views.put("view1", view1);
+
+    Map<String, Object> view_ddoc = new HashMap<>();
+    view_ddoc.put("_id", "_design/name");
+    view_ddoc.put("views", views);
+
+    db.save(view_ddoc);
+    }
+</pre>
+
+<h2 id="Using Views">Using Views</h2>
+
+For understanding the concept of views please refer to the
```

Comment:
reword "For understanding the concept of views" to something like "For background information on..." or similar